### DISCUSSION TITLE
Northbound system has expired

### DISCUSSION THREAD

**Discussion by GlennKarlsson** [tijsverkoyen/HomeAssistant-FusionSolar#149] [Q&A]
Hi,
Thanks for a great HA integration. Got this up and running some weeks ago and last week, all of a sudden, it stopped working. I tried a login-call via Postman and got an error 20003 - northbound system has expired. Anyone know what that is? Does my installer need to fix anything? Can I do it myself? 
regards,
Glenn

**Comment by tijsverkoyen** [comment]
You should contact your installer and ask them to check your credentials.
